src/templates/profile/body/userInfo/UserInfo.js

```diff
@@ -1,40 +1,220 @@
+                                    </Col>
+                                </Row>
+                                <Row className="mt-3">
+                                    <Col>
+                                        ORCID:
+                                    </Col>
+                                </Row>
                             </Col>
+                            {!editMode ?
+                                <Col>
+                                    <Row>
+                                        <Col>
+                                            {userProfile['firstName'] ?
+                                                <> {userProfile['firstName']} </>
+                                                : <> Unknown </>
+                                            }
+                                        </Col>
+                                    </Row>
+                                    <Row className="mt-3">
+                                        <Col>
+                                            {userProfile['lastName'] ?
+                                                <> {userProfile['lastName']} </>
+                                                : <> Unknown </>
+                                            }
+                                        </Col>
+                                    </Row>
+                                    <Row className="mt-3">
+                                        <Col>
+                                            {userProfile['email'] ?
+                                                <> {userProfile['email']} </>
+                                                : <> Unknown </>
+                                            }
+                                        </Col>
+                                    </Row>
+                                    <Row className="mt-3">
+                                        <Col>
+                                            {userProfile['organization'] ?
+                                                <> {userProfile['organization']} </>
+                                                : <> Unknown </>
+                                            }
+                                        </Col>
+                                    </Row>
+                                    <Row className="mt-3">
+                                        <Col>
+                                            {userProfile['orcid'] ?
+                                                <> {userProfile['orcid']} </>
+                                                : <> </>
+                                            }
+                                        </Col>
+                                    </Row>
+                                </Col>
+                                : <Col>
+                                    <Row className="pb-1">
+                                        <Col className={`profile_input`}>
```

Comment:
Implemented Formik framework for handling this form, will be using this for all forms since this is just the perfect way to safe custom code to set form state and stuff